packages/node-client/lib/index.ts

```diff
@@ -705,6 +705,10 @@ export class Nango {
             ...customPrefixedHeaders
         };
 
+        if (customHeaders?.['Content-Type']) {
+            headers['Content-Type'] = customHeaders['Content-Type'];
+        }
+
```

Comment:
without setting the Content-Type, axios below assumes `application/x-www-form-urlencoded` when data is a Buffer